### PR TITLE
Update server-properties-advanced-page-reporting-services.md

### PR DESCRIPTION
**Pull request by cmfinlan**
Added new setting and explanation for "EnableCDNVisuals"

### CODE REVIEW

**Comment by PRMerger18** [comment]
@cmfinlan : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/reporting-services/tools/server-properties-advanced-page-reporting-services.md [hunk 1]
```diff
@@ -105,6 +105,12 @@ Specifies the number of seconds until a report edit session times out. The defau
 ###  EnableClientPrinting  
 Determines whether the RSClientPrint ActiveX control is available for download from the report server. The valid values are **true** and **false**. The default value is **true**. For more information about additional settings that are required for this control, see [Enable and Disable Client-Side Printing for Reporting Services](../../reporting-services/report-server/enable-and-disable-client-side-printing-for-reporting-services.md).  
 
+### EnableCDNVisuals 
+
+(Power BI Report Server only)
+
+When enabled Power BI Reports will load the latest certified custom visuals from a content delivery network (CDN) hosted by Microsoft. If your server does not have access to internet resources you can disable this option and custom visuals will be loaded from the report that was published to the server. *Default is True.*
```
Comment (maggiesMSFT) [inline docs/reporting-services/tools/server-properties-advanced-page-reporting-services.md]:
```suggestion
When enabled, Power BI reports load the latest certified custom visuals from a content delivery network (CDN) hosted by Microsoft. If your server doesn't have access to internet resources, you can turn off this option. In that case, custom visuals are loaded from the report that was published to the server. Default is **True**.  
```

Comment (cmfinlan) [reply docs/reporting-services/tools/server-properties-advanced-page-reporting-services.md]: #signoff

**Comment by maggiesMSFT** [comment]
#sign-off

**Comment by ktoliver** [comment]
Hi @maggiesMSFT 

I'm not able to edit the article, for some reason--the **Edit** button in the diff view is grayed out, and I get a permissions error if I try to commit your suggested revisions. It looks like we'll need to have the contributor update his branch with your revisions, or we can move the commit to the private repo, where I can make the edits. Which do you prefer?

**Comment by maggiesMSFT** [comment]
Thanks, @ktoliver -- same for me. I'll just start over in a private branch. I want to make a few small changes to other articles anyway.

**Comment by maggiesMSFT** [comment]
changes in https://github.com/MicrosoftDocs/sql-docs-pr/pull/13514

**Comment by maggiesMSFT** [comment]
#please-close

**Pull request closed without merging**